dashboard/models.py

```diff
@@ -172,17 +172,22 @@ def get_data_last_updated(self) -> Optional[datetime]:
         """ Returns the datetime of the last cron run of all courses
```

Comment:
This method name may not be appropriate anymore, since you're using `earliest_start_datetime` inside of it (which has nothing to do with getting the `data_last_updated` value. I guess I'd argue there should be some higher-level method (`determine_resource_data_filter`?) that uses the results of `earliest_start_datetime` and `get_data_last_updated` to figure out what date should be used to filter.

Either that, or rename this method?

Reply:
When I was fixing this I noticed that using latest instead of earliest was incorrect since we'd need to go back to the earliest updated course. But since all the dates were the same it *really* didn't matter at the moment. 

I can change this method name here to be consistent with this.

Reply:
Renamed this method to `get_data_earliest_date`